### PR TITLE
Added rockspec

### PR DESCRIPTION
**Pull request by greatwolf**

### CODE REVIEW

File: rockspec/rxlua-scm-2.rockspec [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  summary = "Reactive Extensions for Lua",
+  homepage = "https://github.com/bjornbytes/RxLua/tree/master",
+  license = "MIT/X11",
+  maintainer = "tie.372@gmail.com",
+  detailed = [[
+RxLua gives Lua the power of Observables, which are data structures that represent a stream of values that arrive over time. They're very handy when dealing with events, streams of data, asynchronous requests, and concurrency.
+]]
+}
+
+dependencies = { "lua >= 5.1" }
```
Comment (bjornbytes) [inline rockspec/rxlua-scm-2.rockspec]: Is there a reason the `dependencies` section is only in `scm-2`?

**Comment by bjornbytes** [comment]
Hey, thanks for getting RxLua on luarocks!  I may make some cosmetic changes to the rockspec files if that's alright with you (single quotes for strings and wrapping of long lines), and I'd like the `maintainer` field to be "https://github.com/bjornbytes", but other than that everything looks good.

**Comment by greatwolf** [comment]
yeah, feel free to make any changes to conform to the guidelines.

File: rockspec/rxlua-scm-2.rockspec (hunk 1 above)
Comment (greatwolf) [inline rockspec/rxlua-scm-2.rockspec]: I wasn't sure whether to make the changes retroactive or not. It didn't seem like rockspec of other lua projects did that. But you can certainly add that change in, naming it like `rxlua-0.0.1-2.rockspec` and `rxlua-0.0.2-2.rockspec`.

**Pull request merged**